indico/modules/search/client/js/components/SearchApp.jsx

```diff
@@ -175,10 +176,49 @@ ResultHeader.defaultProps = {
   categoryTitle: undefined,
 };
 
+function SearchOptions({sort, sortOptions, onSortChange}) {
+  if (!sortOptions.length) {
+    return null;
+  }
+
+  const selected = sortOptions.find(x => x.key === sort) || sortOptions[0];
+
+  return (
+    <div styleName="search-options">
+      <Dropdown
+        text={Translate.string('Sort by: {value}', {value: selected.label})}
```

Comment:
Probably a matter of preference, but just putting `text={value}` here doesn't look bad either:

![image](https://user-images.githubusercontent.com/179599/117450620-c2bf8b00-af41-11eb-950e-d8cfa34b752c.png)

Reply:
I like you idea but it seems less user-friendly and more hidden from the regular user.